volume-group-operator/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/example/memcached-operator
```

Comment:
can you change `github.com/example/memcached-operator` to `github.com/IBM/volume-group-operator`? and change all the other appearances of `memcached` accordingly?

Reply:
done